### PR TITLE
[DOC] Fixed a typo in transformer extension template

### PR DESCRIPTION
**Pull request by rakshitha123**
#### What does this implement/fix? Explain your changes.

Fixed a spelling mistake.

##### For all contributions
- [x] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1431,6 +1431,15 @@
         "code"
       ]
     },
+    {
+      "login": "rakshitha123",
+      "name": "Rakshitha Godahewa",
+      "avatar_url": "https://avatars.githubusercontent.com/u/7654679?v=4",
+      "profile": "https://github.com/rakshitha123",
+      "contributions": [
+        "code"
```
Comment (fkiraly) [inline .all-contributorsrc:1440]: you haven't contributed any code yet! (that's still to come 😃 )

Fixing typo in docs is a `"docs"` contribution.

**Review by fkiraly — CHANGES_REQUESTED** [review]
Looks good!

Change request, kindly change your contribution to a `"doc"` contribution.

(usually we're not that picky... I just thought this would be a good way to get you acquainted with the PR review system on GitHub. You now need to make the requested change, update your PR, answer all outstanding discussions, and re-request a review once it's done.)

File: .all-contributorsrc (hunk 1 above)
Comment (rakshitha123) [reply .all-contributorsrc]: Changed the contribution type to docs

**Review by fkiraly — APPROVED** [review]
Excellent, thanks!

**Review by fkiraly — CHANGES_REQUESTED** [review]
ok, actually, I think it should be "doc", not "docs"?

**Review by fkiraly — APPROVED** [review]
Alright, great!

**Pull request merged**